DigitalExcellence/dex-backend
# Feature/265 data officer role

## Description

Created the new data officer role and created CRUD functionality for institutions.

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Check the wiki and check the permissions page. This page should be upaded and should contain the new data officer role and the new endoints for the institution controller. Also, the existing endpoints that can be reachable for data officer should be updated.
2. Run the postman tests and check that no test fails (Response time is less than ....ms tests excluded).
3. Run the InstitutionServiceTests and check that no test fails.
4. Run the InstitutionRepositoryTests and check that no test fails.

## Link to issue

Closes: #265
